nautobot/extras/querysets.py

```diff
@@ -158,7 +158,15 @@ def _get_config_context_filters(self):
 
 
 class DynamicGroupQuerySet(RestrictedQuerySet):
-    """Queryset for `DynamicGroup` objects that provides a `get_for_object` method."""
+    """Queryset for `DynamicGroup` objects that provides `get_for_object` and `get_for_model` methods."""
+
+    def get_for_model(self, model):
+        """
+        Return all DynamicGroups assignable to the given model class.
+        """
+        concrete_model = model._meta.concrete_model
+        content_type = ContentType.objects.get_for_model(concrete_model)
```

Comment:
Why is concrete_model required here?

Reply:
Not sure it is needed, tbh - was just following the pattern in custom field, computed field, and relationship managers/querysets.